content/applications/finance/accounting/fiscal_localizations/localizations/kenya.rst

```diff
@@ -0,0 +1,119 @@
+     - Installing this module integrates with the Kenyan G03 Tremol control unit device to report
+       taxes to KRA through TIMS.
+
+.. image:: kenya/modules.png
+   :align: center
+   :alt: The three modules for the Kenya Fiscal Localization Package on Odoo
+
+Kenyan TIMS integration
+=======================
+
+The Kenya Revenue Authority (KRA) has decided to go digital for tax collection through the **Tax
+Invoice Management System (TIMS)**. As of December 1st, 2022, all VAT-registered persons should
+comply with TIMS. The goal is to reduce VAT fraud, increase tax revenue, and increase VAT compliance
+through standardization, validation, and transmission of invoices to KRA on a real-time or near
+real-time basis.
+
+All VAT-registered taxpayers should use a **compliant tax register**. Odoo decided to develop the
+integration of the **Tremol G03 Control Unit (type C)**, which can be run locally through USB. This
+device validates invoices to ensure financial documents meet the new regulations and send the
+validated tax invoices directly to KRA. Installing a proxy server that provides a gateway between
+users and the internet is required.
+
+Installing the proxy server on a Windows device
+-----------------------------------------------
+
+Go to odoo.com/download, fill out the required information and click :guilabel:`Download`.
+
+.. image:: kenya/download.png
+   :align: center
+   :alt: Install the Proxy Server on a Windows device
+
+Once it is loaded on your computer, a wizard opens. You have to read and agree with the terms of the
+agreement. On the next page, select the :guilabel:`type of install: Odoo IoT. Then, click
+:guilabel:`Next` and :guilabel:`Install`. Once completed, click :guilabel:`Next`. Check the
+:guilabel:`Start Odoo` box to be redirected to Odoo automatically, and then click :guilabel:`Finish`.
+
+A new page opens, confirming your :doc:`IoT Box <../../../../productivity/iot/config/connect>` is up
+and running. Connect your physical device **Tremol G03 Control Unit (type C)** to your laptop via
+USB. In the :guilabel:`IoT Device` section, check that your Tremol G03 Control Unit (type C) appears.
+
+.. image:: kenya/iot-box.png
+   :align: center
+   :alt: Your IoT box is up and running
+
+.. note::
+   If the device is not detected, try to plug it in again or click on the :guilabel:`Restart` button
+   in the top right corner.
+
+.. seealso::
+   :doc:`Connect an IoT box to your database <../../../../productivity/iot/config/connect>`
+
+Sending the data to KRA using the Tremol G03 Control Unit
+---------------------------------------------------------
+
+As a pre-requisite, check out that the :ref:`Kenyan Accounting modules
+<localization/kenya/configuration>` are installed on your database. Then, go to
+:menuselection:`Accounting --> Configuration --> Settings --> Kenya TIMS Integration section`, and
+check that the :guilabel:`control Unit Proxy Address` has been automatically filled out, confirming
```

Comment:
This is not strictly true. It's just that the default value of the control unit proxy address in settings is `http://localhost:8069` and in this example it was the correct address. It won't automatically change to the right address (even if the iot box is configured correctly). 

It's important that the address should match that of the IoT box.
In the example from the video, the IoT box server is running on the computer, so it has a local address (hence, `http://localhost:8069` is working by default). It also has an address on the local area network (hence, in this example, `196.168.0.13:8069` should also work in the control unit proxy address field in settings too). But odoo does not automatically detect the address at the moment.

The best way of confirming the connection between the device and your computer is that IoT box server page (where at the bottom it says `IOT Device` > `Fiscal data modules` > `Tremol G03 Serial: COM3`

Reply:
Updated, thanks :)